### ISSUE TITLE
Proposal: add a walkthrough for running Athens from the docker image?

### ISSUE THREAD

**Issue by arschles** [gomods/athens#601]
We have a [walkthrough](http://docs.gomods.io/walkthrough/) that explains how to use go modules without the proxy, how to build the proxy, and how to run the proxy locally. We also have documentation on how to deploy a [shared team instance](http://docs.gomods.io/install/shared-team-instance/) that explains how to deploy Athens with docker.

I feel like it would be nice to have a walkthrough that explains how to get started with Docker, instead of building and running locally.

**Comment by kaxcode** [comment]
Would this be a combination of the [shared team instance](https://docs.gomods.io/install/shared-team-instance/) and [walkthrough](https://docs.gomods.io/walkthrough/) documentation?

**Comment by robjloranger** [comment]
I _think_ the shared team doc is already doing this. It never goes into how to deploy, that part is being worked on afaik.

I think once #81 is complete, we could use that image in the docs.

**Comment by kaxcode** [comment]
I went through the shared team doc and it does go through how to use the proxy with Docker really well. Maybe renaming the shared team doc to ‘walkthrough with docker’ would help.

**Comment by robjloranger** [comment]
I like that, and maybe we could keep the original title and create a short doc that outlines the series with links to each one. Starting with the Docker walkthrough.

**Comment by kaxcode** [comment]
I'm going through the shared team doc again. This doc still assumes the user has Go installed locally. Should this be changed to run everything on Docker? Is that doable?
I deleted Go from my computer to test this. I go this error when trying to run the `go run .` command on the walkthrough repo. 
<img width="764" alt="screen shot 2018-09-02 at 9 10 58 pm" src="https://user-images.githubusercontent.com/19161242/44962986-cd096b80-aef4-11e8-8feb-daa67a6aa3e2.png">

**Comment by arschles** [comment]
Closing in favor of #420
